src/Peering/Peering/help/New-AzPeering.md

```diff
@@ -0,0 +1,342 @@
+```
+
+### -PeerAsnResourceId
+The Peer Asn Resource Id. Use Get-AzPeerAsn to retrieve the Id.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 3
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PeeringLocation
+The Physical Location Different from Azure Region.
+Use Get-AzPeeringLocation -Kind \<kind\> use City name as key.
+
+```yaml
+Type: System.String
+Parameter Sets: Exchange, Direct
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+```yaml
+Type: System.String
+Parameter Sets: ParameterSetNameConvertLegacyPeering
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+The resource group name.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Tag
+The tags to associate with the Microsoft Peering Service.
+
+```yaml
+Type: System.Collections.Hashtable
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -UseForPeeringService
+Enable for use with Microsoft Peering Service (MPS).
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: Direct
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Confirm
+Prompts you for confirmation before running the cmdlet.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases: cf
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -WhatIf
+Shows what would happen if the cmdlet runs. The cmdlet is not run.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases: wi
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
+
+### Microsoft.Azure.PowerShell.Cmdlets.Peering.Models.PSPeering
+
+### System.String
+
+### System.Management.Automation.SwitchParameter
+
+### System.Collections.Generic.List`1[[Microsoft.Azure.PowerShell.Cmdlets.Peering.Models.PSExchangeConnection, Microsoft.Azure.PowerShell.Cmdlets.Peering, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null]]
```

Comment:
Please fix this by running Update-MarkdownHelpModule with the -UpdateInputOutput flag.